### ISSUE TITLE
Extension has been removed from Chrome Web Store

### ISSUE THREAD

**Issue by AlessandroFC15** [AlessandroFC15/Fuck-Youtube#34]
Well guys, looks like it was good while it lasted on Chrome. 

The Google Support team just sent an email informing that the item was removed from the Google Chrome Web Store.

I'm trying to get in touch with them as soon as possible. 

If anyone still has the item installed, enjoy it and don't uninstall it, because you might not have a chance to get it back :)

**Comment by seanbperiod** [comment]
Can this still be downloaded from here and installed via dev mode?

**Comment by AlessandroFC15** [comment]
@seanbperiod Oh, that's good news. So at least we can install it like that. We are currently working on others ways to offer the extension on Chrome apart from the Web Store.

**Comment by IkeKap** [comment]
Another option would be to add it to the mozilla addon store. They are notably more lax when it comes to extensions published on their store as long as they aren't malicious and are supported (IIRC, double check just in case)

**Comment by AlessandroFC15** [comment]
@IkeKap Yes, absolutely, that was the first thing that crossed my mind. I've worked today on porting the extension to Firefox and just published it on their store. 

You can find it right here: https://addons.mozilla.org/pt-BR/firefox/addon/f-ck-youtube/

It is in an early access stage, Mozilla calls it the "experimental" phase, I haven't post anywhere yet. If anybody can give any feedback regarding if the extension is working correctly on Firefox as well, that would be great. It worked for me.

Thanks for your input :)

**Comment by kudlatyamroth** [comment]
i think it show only for you, as long as it is an experimental phase, or for someone that you invite to test

**Comment by AlessandroFC15** [comment]
@kudlatyamroth I've set the status of the extension to invisible for now, while I work on some issues that were happening on Firefox. It should be available again soon.
